setup.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+"""Project Setup"""
+from setuptools import find_packages, setup
+
+setup(
+    name='p8tool',
```

Comment:
`picotool` differs from the top-level package name (`p8tool`) and the main command-line entry point (`p8tool`) so I chose (`p8tool`) here.

Reply:
At some point, it may make sense to unify that topmost token so that someone can `pip install FOO` then `import FOO` for the same `FOO`.